api/v1alpha1/nifidataflow_types.go

```diff
@@ -30,6 +30,8 @@ type NifiDataflowSpec struct {
 	RegistryClientRef *RegistryClientReference `json:"registryClientRef,omitempty"`
 	// describes the way the operator will deal with data when a dataflow will be updated: drop or drain
 	UpdateStrategy ComponentUpdateStrategy `json:"updateStrategy"`
+	// contains the reference to the NifiResource which is the Parent Process Group for this dataflow
+	ParentProcessGroupReference *ResourceReference `json:"parentProcessGroupRef,omitempty"`
```

Comment:
Should be removed.

`NifiDataflow` is in `v1`, the `v1alpha1`must not be updated.